### DISCUSSION TITLE
blog/what-is-new-in-istio-1-13/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#291] [Announcements]
# blog/what-is-new-in-istio-1-13/

2022 年 2 月 Istio 发布 Istio 1.13.0 和 Istio 1.13.1，这篇博客将想你介绍这两个版本中有哪些值得注意的新特性。

https://jimmysong.io/blog/what-is-new-in-istio-1-13/

**Comment by txlong2012slug** [comment]
Istio 1.13 支持 Kubernetes 1.20、1.21、1.22 和 1.23 版本，并在 Kubernetes 1.16、1.17、1.18、1.19 中测试过，**但并得到官方支持**。

这句话表示支持还是不支持？ 不支持的话是因为什么呢？

**Reply by Shulong123** [reply]
我看官方写的是Tested, but not supported
